compiler/fir/checkers/src/org/jetbrains/kotlin/fir/analysis/checkers/expression/DefaultExpressionCheckers.kt

```diff
@@ -14,7 +14,7 @@ object CommonExpressionCheckers : ExpressionCheckers() {
         FirAbstractSuperCallChecker,
         FirQualifiedSupertypeExtendedByOtherSupertypeChecker,
         FirProjectionsOnNonClassTypeArgumentChecker,
-        FirUpperBoundViolatedChecker,
+        FirUpperBoundViolatedChecker
```

Comment:
Please do not edit side files without a reason, especially when you are not the person who mostly works with this side file.